railties/test/generators/app_generator_test.rb

```diff
@@ -698,6 +698,17 @@ def test_kamal_files_are_skipped_if_required
     assert_no_file ".kamal/secrets"
   end
 
+  def test_database_yml_skip_kamal
+    # Regression test for #54549
```

Comment:
I think git history will usually indicate why a change was made so I'm not sure the comment is necessary